packages/next-codemod/package.json

```diff
@@ -20,5 +20,8 @@
     "build": "yarn tsc -d -w -p tsconfig.json",
     "test": "jest"
   },
-  "bin": "./bin/next-codemod.js"
+  "bin": "./bin/next-codemod.js",
+  "devDependencies": {
+    "typescript": "3.8.3"
```

Comment:
Since this workspace uses `tsc` in its `scripts` field, it needs to declare it as a dependency (even without PnP). And whenever you want to upgrade tsc in the future, just run `yarn up typescript`, which will automatically upgrade it across all workspaces that use it.

Reply:
As an alternative you could also use `yarn run -T tsc` to use the typescript version from the root